pkg/rpc/response/errors.go

```diff
@@ -8,13 +8,17 @@ import (
 )
 
 type (
-	// Error object for outputting JSON-RPC 2.0
-	// errors.
-	Error struct {
-		Code     int64  `json:"code"`
-		HTTPCode int    `json:"-"`
-		Message  string `json:"message"`
-		Data     error  `json:"data,omitempty"`
+	// ServerError object for outputting JSON-RPC 2.0 errors on the server side.
+	ServerError struct {
```

Comment:
And this one actually can easily live somewhere in the server internals, clients don't care anyway.

Reply:
Fixed.